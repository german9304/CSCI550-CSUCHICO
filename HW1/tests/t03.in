20
0 1
1 8
2 4
2 11
3 12
3 15
3 18
4 8
5 7
5 9
5 12
5 14
6 13
6 16
10 19
11 12
13 15
14 19
15 17
