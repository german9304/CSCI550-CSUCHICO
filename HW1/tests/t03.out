11 2 4 8 1 0 
11 2 4 8 1 
11 2 
11 12 3 
11 2 4 
11 12 5 
11 12 3 15 13 6 
11 12 5 7 
11 2 4 8 
11 12 5 9 
11 12 5 14 19 10 
11 12 
11 12 3 15 13 
11 12 5 14 
11 12 3 15 
11 12 3 15 13 6 16 
11 12 3 15 17 
11 12 3 18 
11 12 5 14 19 
